desk/app/grouper.hoon

```diff
@@ -126,8 +126,34 @@
       `this(outstanding-pokes (~(del in outstanding-pokes) [src.bowl i.t.t.wire]))
     ==
   ?-  -.sign
-      %poke-ack   `this
+      %poke-ack
+    ?.  ?=([%dm ship=@ token=@ ~] wire)
+      ?~  p.sign  `this
+      %-  (slog leaf/"Poke failed {<wire>}" u.p.sign)
+      `this
+    ?>  ?=([%dm ship=@ token=@ ~] wire)
```

Comment:
The `?>` is superfluous, since the `?.` already checks for the same condition, no?